src/docs/product/accounts/quotas/org-stats/index.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "Org Stats"
+sidebar_order: 20
+description: "Learn about how events are being used across your organization."
+---
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The **Organization Stats** page allows you to check your organization's event usage while also providing a high-level overview of how different projects are using Sentry. The page reflects events the system has received across your entire organization along with a list of all projects and their consumption of errors, transactions, and attachments. You can see which projects have had a recent spike or are the most active and may need more attention. On this page, you can also control the date range that’s displayed, allowing you to focus on a smaller period of time or zoom out for a longer view.
+
+The page is designed for org admins, but is available to all team members. This is helpful if a team member who isn't an admin is tasked with reviewing these statistics. Also, this allows team members associated with specific projects to use this page to investigate why their events are being dropped.
```

Comment:
```suggestion
The **Organization Stats** page allows you to check your organization's event usage while also providing a high-level overview of how different projects are using Sentry. The page reflects events the system has received across your entire organization along with a list of all projects and their consumption of errors, transactions, and attachments. Events fall into one of three categories: *Accepted*, *Filtered*, or *Dropped*. Filtered events are events that were blocked due to your [inbound data filter](/product/data-management-settings/filtering/) rules. Dropped events are ones the system discarded due to rate limits, quotas, spike protection, or because of invalid data.

You can see which projects have had a recent spike or are the most active and may need more attention. On this page, you can also control the date range that’s displayed, allowing you to focus on a smaller period of time or zoom out for a longer view. The page is designed for org admins, but is available to all team members. This is helpful if a team member who isn't an admin is tasked with reviewing these statistics. Also, this allows team members associated with specific projects to use this page to investigate why their events are being dropped.
```